### PR TITLE
Pusheen command

### PR DESCRIPTION
**Pull request by metehus**
Pusheen command /ᐠ｡ꞈ｡ᐟ\ uwu

### CODE REVIEW

File: src/commands/misc/pusheen.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      '=^._.^= ∫',
+      '(^._.^)= ﾉ',
+      '(^・x・^)',
+      '(=♡ ᆺ ♡=)',
+      '(=◕ᆽ◕ฺ=)'
+    ]
+    const posts = await this.client.apis.tumblr.getPhotoPosts('pusheen.com')
+    const randomPost = getRandom(posts.response.posts)
+    const embed = new SwitchbladeEmbed(author)
+      .setAuthor(posts.response.blog.title, `https://i.imgur.com/${getRandom(pusheenAvatars)}.png`, (randomPost).post_url)
```
Comment (davipatury) [inline src/commands/misc/pusheen.js]: why use () in randomPost

Comment (metehus) [reply src/commands/misc/pusheen.js]: lmao idk

File: src/commands/misc/pusheen.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+
+module.exports = class Pusheen extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'pusheen'
+    this.category = 'general'
+  }
```
Comment (davipatury) [inline src/commands/misc/pusheen.js]: `this.requirements = new CommandRequirements(this, { apis: ['tumblr'] })`

**Comment by pedrofracassi** [comment]
Let's do it!

**Pull request merged**